safe_cli/main.py

```diff
@@ -118,11 +122,39 @@ def loop(self):
                 pass
 
 
-def build_safe_cli():
+def get_safe_from_owner(
```

Comment:
For me this doesn't belong here, maybe in `utils` as it's similar to `get_erc_20_list` (btw, we should remove the # comments for that function and add proper Python docs)?

Reply:
You say comments in the utils files?